RELEASE.md

```diff
@@ -267,21 +278,22 @@ should point the user to the Qubes notifications in the top-right corner:
     https://developer.apple.com and login with the proper Apple ID.
 
 #### Releasing and Signing
-- [ ] Verify and install the latest supported Python version from [python.org](https://www.python.org/downloads/macos/)
+
+- [ ] Verify and install the latest supported Python version from [python.org](https://www.python.org/downloads/macos/) (do not use the one from brew as it is known to cause issues)
+  * In case of a new install, reinstall poetry with `python3 -m pip install poetry`
```

Comment:
```suggestion
  * In case of a new Python installation or minor version upgrade, e.g., from 3.11 to 3.12 , reinstall Poetry with `python3 -m pip install poetry`
```